sockets_test.go

```diff
@@ -19,7 +20,9 @@ const (
 	sendPath        string = "/sender"
 	pingPath        string = "/ping"
 	recvStringsPath string = "/strings/receiver"
+	recvBinaryPath  string = "/binary/receiver"
```

Comment:
As per the other comments, this should be something like `"/byteslice/receiver"`